README.md

```diff
@@ -18,6 +18,7 @@ To make a retro styled game, the whole process of creation and execution takes p
 - Multiple programming languages: [Lua](https://www.lua.org),
   [Moonscript](https://moonscript.org),
   [Javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript),
+  [Ruby](https://www.ruby-lang.org/en/),
```

Comment:
I think this should be https://mruby.org

MRuby and Ruby are NOT quite the same project.  It's certainly possible to embed Ruby also, but since we're not doing that we should be clear that we're using Ruby, at the very _least_ the link here should point to MRuby I'd think.

Reply:
Yes, I've been thinking of that too, there is some source-level incompatibility between the two, but at the same time we shouldn't create any more confusion to the end users.

IMO it's something worth mentioning in the documentation.

Reply:
> we shouldn't create any more confusion to the end users.

I think `[Ruby](https://mruby.org)` would solve the issue, no?

Reply:
> I think [Ruby](https://mruby.org) would solve the issue, no?

Ruby is the language, MRuby is just one of the available implementations.

Just like with Javascript (or should I say ECMAscript); duktape isn't perfectly compliant either:

> ```markdown
>    [Javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript), 
> ```